src/components/InteractiveArticle.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import BaseComponent from 'lib/falcor/FalcorController';
+
+export default class InteractiveArticle extends FalcorController {
+  static getFalcorPaths(params) {
+    return [
+      // For now we only use the html part, the js and css parts are for further improvements
+      ['interactiveBySlug', params.slug, ['html', 'js', 'css']],
+    ];
+  }
+
+  render() {
+    const slug = this.props.params.articleSlug;
+    const reactHtml = {
+      __html: this.state.data.interactiveBySlug[slug].html,
+    };
+    return <div dangerouslySetInnerHTML={reactHtml} />;
```

Comment:
I'm a little confused with what you're suggesting. React Helmet only changes the site's metadata, not the actual navbar that displays at the top of the page.

Wouldn't just injecting the HTML mean that we still have the same standard navigation bar at the top of the page?